### PR TITLE
Add support caja

### PR DESCRIPTION
**Pull request by ericpre**
- [x] Add support for caja file manager,
- [x] add option to select a file manager,
- [x] remove unnecessary dependencies,
- [x] update readme,
- [x] ready for review.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -99,40 +94,46 @@ Or install from conda-forge channel using conda (in a Anaconda/Miniconda distrib
...
-~~~~~~~~~~~~~~~~~
+Optional parameters
+~~~~~~~~~~~~~~~~~~~
+
+On Linux, several file manager can be installed, to create or remove the context
+menu shortcut(s) for a specific file manager, use the ``--file_manager`` (``-f``) option:
+
+.. code:: bash
+
+    $ start_jupyter_cm -f nautilus
```
Comment (francisco-dlp) [inline README.rst:135]: ``nautilus`` -> ``Nautilus``?

Comment (ericpre) [reply README.rst:135]: Or support both?

File: start_jupyter_cm/command.py [hunk 2]
```diff
@@ -6,24 +6,28 @@
 parser = argparse.ArgumentParser(
     description='Add or remove context menu entries to start Jupyter applications in a given directory')
 parser.add_argument('--remove', action="store_true", help='remove the entries')
+parser.add_argument('-f', '--file_manager', action="store", help='select a specific file manager (Linux only)')
```
Comment (francisco-dlp) [inline start_jupyter_cm/command.py:9]: Maybe explain in the help which are the valid options?

File: README.rst (hunk 1 above)
Comment (francisco-dlp) [reply README.rst:135]: I would go for supporting just the low-case version. Most arguments elsewhere are case-sensitive, so we shouldn't spoil our users ;)

Comment (ericpre) [reply README.rst:135]: Yes, agree done!

**Pull request merged**